components/Block.vue

```diff
@@ -75,66 +74,57 @@ export default {
     },
     handleDragStart(e) {
       const { clientX, clientY, offsetX, offsetY } = e
-      const { offsetHeight, offsetWidth } = e.target.offsetParent.parentElement
+      const {
+        height,
+        width
+      } = e.target.parentNode.parentNode.getBoundingClientRect()
```

Comment:
I don't know what the DOM structure looks like but `parentNode.parentNode` seems brittle. Is there a more direct reference you can get?

Reply:
I don't think there's a better way to get the grandparent through the event, I could just throw an ID on the parent and grab it that way with `.getElementById(id)` I guess